create-oci.sh

```diff
@@ -114,8 +155,31 @@ if [ ${#artifacts[@]} != 0 ]; then
     source oras_opts.sh
 
     pushd "${archive_dir}" > /dev/null
-    oras push "${oras_opts[@]}" --registry-config <(select-oci-auth.sh ${repo}) "${store}" --config="${config:-}" \
-        "${artifacts[@]}"
+    if [ -z "${attach}" ]; then
+        oras push "${oras_opts[@]}" --registry-config <(select-oci-auth.sh ${repo}) "${store}" --config="${config:-}" \
+            "${artifacts[@]}"
+    else
+        oci_artifact_type="application/vnd.konflux-ci.trusted-artifact"
+        if [ -n "${oci_type_scope}" ]; then
+            oci_artifact_type="${oci_artifact_type}.${oci_type_scope}"
+        fi
+        attached_artifacts=()
+        for artifact in "${artifacts[@]}"; do
+            file_base="${artifact%.*}"
+            file_extension="${artifact##*.}"
+            media_type_descriptor="${file_base}"
+            if [[ "${file_base}" != "${file_extension}" ]]; then
+                media_type_descriptor="${media_type_descriptor}+${file_extension}"
+            fi
+            echo "registering artifact:"
+            echo "${artifact}:${oci_artifact_type}.${media_type_descriptor}"
+            attached_artifacts+=("${artifact}:${oci_artifact_type}.${media_type_descriptor}")
+        done
+        oras attach "${oras_opts[@]}" --no-tty --registry-config <(select-oci-auth.sh ${repo}) --artifact-type "${oci_artifact_type}" \
+            --distribution-spec v1.1-referrers-api "${store}" "${attached_artifacts[@]}"
+        oras attach "${oras_opts[@]}" --no-tty --registry-config <(select-oci-auth.sh ${repo}) --artifact-type "${oci_artifact_type}" \
+            --distribution-spec v1.1-referrers-tag "${store}" "${attached_artifacts[@]}"
```

Comment:
No need to force both options, ORAS will pick the best one available
```suggestion
        oras attach "${oras_opts[@]}" --no-tty --registry-config <(select-oci-auth.sh ${repo}) --artifact-type "${oci_artifact_type}" "${store}" "${attached_artifacts[@]}"
```